### DISCUSSION TITLE
Skip payment methods after booking

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#582] [Q&A]
Hi,
I want to Skip payment methods after booking. It is possible to set this by editing the single event in the backend, tab RSVP/Payment method.
But I would like to set this by default for all my events, and I don't find the way to. Is it possible ?

**Comment by f14m07** [comment]
I have the solution of a filter hook but maybe another way? 
```
$event['event_properties']['skippaymentoptions'] = 1;
```

**Reply by liedekef** [reply]
this is indeed the wordpress method of doing things, so yes: currently a filter is your only option here
